### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by GB-sv** [OpenBoard-org/OpenBoard#1149]
Can Openboard run on Windows on Arm because I have display problems on my surface pro 11.?

**Comment by Vekhir** [comment]
Please consider your use of punctuation marks, one is usually enough.

Are you running it on Windows on ARM and experience issues with it? Then please detail the issues because it apparently runs.
In any case, x86 applications are emulated on Windows on ARM, so could be an issue with that, though unlikely.

**Comment by GB-sv** [comment]
> Please consider your use of punctuation marks, one is usually enough.
> 
> 
> 
> Are you running it on Windows on ARM and experience issues with it? Then please detail the issues because it apparently runs.
> 
> In any case, x86 applications are emulated on Windows on ARM, so could be an issue with that, though unlikely.

Yes I am running it on Windows on Arm and experience issues with it.
I think it's probably because it is emulated. 
Here are some photos that show that some menus are not visible and very small : ![image](https://github.com/user-attachments/assets/ed59228e-cf1b-4b2e-805a-f99e6005134f)
